paraoleg/lab1-BasicIO/main.c

```diff
@@ -0,0 +1,31 @@
+/******************************************************************************/
+/*  Files to Include                                                          */
+/******************************************************************************/
+#ifdef __XC32
+    #include <xc.h>          /* Defines special function registers, CP0 regs  */
+#endif
+
+#include <stdint.h>         /* For uint32_t definition                        */
+#include <stdbool.h>        /* For true/false definition                      */
+
+#include "system.h"         /* System funct/params, like osc/periph config    */
+#include "user.h"           /* User funct/params, such as InitApp             */
+
+/******************************************************************************/
+/* Global Variable Declaration                                                */
+/******************************************************************************/
+
+/* i.e. uint32_t <variable_name>; */
+
+/******************************************************************************/
+/* Main Program                                                               */
+/******************************************************************************/
+
+int32_t main(void)
```

Comment:
Either make return type `void` or provide the `return 0` statement (would be unreachable)